calcom/cal.com
# feat: Add 'monobot' web application into cal.com app-store.

## What does this PR do?



- This pool request includes adding a 'monobot' web application to "packages/app-store" directory, in accordance with the all yours requirements.



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- "Monobot.ai" is production ready application. With it you can create AI Agents that Work as a Customer Service.
- Last "Monobot.ai" updates includes cal.com integration.
- Our AI agents can use the "cal.com" service to create/update/delete calendar events.
- In order to check cal.com integration:
   1. Sign In to "monobot.ai" app. (https://monobot.ai/sign-in) (username: monobot.acceptance2@gmail.com, password: password2)
   3. Proceed to "AI Agents" list. 
   4. Proceed to "Barber Shop" ai assistant configuration. (AI assistant with name: "Barber Shop" already created for the tests)
   5. Proceed to Tool tab.
   6. Current ai assistant already have preconfigured tools for various purposes related to cal.com.
   7. All that remains to be done is add an action configuration mandatory needs: **"API Key", "Event Type ID"**

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings
